environ/environ.go

```diff
@@ -79,6 +79,7 @@ func Stage(stage *drone.Stage) map[string]string {
 		"DRONE_STAGE_STARTED":    fmt.Sprint(stage.Started),
 		"DRONE_STAGE_FINISHED":   fmt.Sprint(stage.Stopped),
 		"DRONE_STAGE_DEPENDS_ON": strings.Join(stage.DependsOn, ","),
+		"DRONE_CARD_PATH":        "/dev/stdout",
```

Comment:
should this not be in the step func ?

Reply:
I checked this with Brad and he said here was the best place.